app/assets/javascripts/scripts.js

```diff
@@ -1,6 +1,40 @@
 +function () {
   var editor = null
 
+  urlQueryParams = function() {
+    var paramsList = window.location.search.substr(1).split('&')
+    var params = {}
+
+    if (paramsList == '')
```

Comment:
Este no me di cuenta de chillar para que fuera:

```suggestion
    if (! paramsList)
```

Lo amás.

Reply:
El tema es que no da lo mismo =P 
```
!paramsList
#=> false
paramsList == ''
#=> true
```

Reply:
Yo creo que no probamos bien lo que pusimos en el comentario ;)

![captura de pantalla de 2018-11-21 14-38-15](https://user-images.githubusercontent.com/85371/48858816-24540500-ed9b-11e8-90c5-1d9060f9504e.png)

Reply:
Ejecuta primero  `paramsList = window.location.search.substr(1).split('&')` jaja es un array con un string adentro =P